Automattic/wc-calypso-bridge
# Remove setting demo store option on ecommerce sites by default

### Changes proposed in this Pull Request:

Closes #1515.

PR removes setting `woocommerce_demo_store` option when ecommerce sites are set up. I assume the demo store notice is no longer relevant (it was implemented [6 years ago](https://github.com/Automattic/wc-calypso-bridge/pull/189/files#diff-574140c5ebef5f5a93893e45669a487fc8f736639e6eb3b19f0869e1735c95beR72)).

<img width="1035" alt="image" src="https://github.com/user-attachments/assets/0e8f905c-118c-465b-85c3-8585d04c87d5">

### How to test the changes in this Pull Request:




1. Use a WPCOM atomic e-commerce site
1. Open up `/_cli` in a new tab
3. Run `option set woocommerce_demo_store no`
2. Run `option delete wpcom_ec_plan_theme_defaults`
3. Go to frontend homepage of the site
4. Refresh at least once to ensure [theme setup](https://github.com/Automattic/wc-calypso-bridge/blob/08bcbba3595cb9ad7b3b6d0db27996885230c699/includes/class-wc-calypso-bridge-themes-setup.php#L92-L109) is run
5. Observe the notice store notice is not shown at the bottom of the page
6. Run `option get wpcom_ec_plan_theme_defaults` and observe the value is `1`





### Other information:




### FOR PR REVIEWER ONLY:
